### DISCUSSION TITLE
Adding a search functionality to label trees page

### DISCUSSION THREAD

**Discussion by agnanas** [biigle/core#971] [Ideas]
When in the label tree page in Biigle, it would be very helpful to have a search functionality there to open up directly to that label like we have in Largo for example. 
I sometimes find myself needing to edit some labels and when I open up the label tree, having to click through the hierarchy to get to my label is tedious.

**Comment by mzur** [comment]
You can do this in the "new label" section. When you type a label name and select a label with the "label parent" input, then the label is selected in the label tree on the left as well.

This is not really obvious, though. Do you think we should add a separate "search" input that basically does the same thing but is easier to find?

**Reply by agnanas** [reply]
Oh that is fantastic. That will do it. Thanks a lot!
Don't think I need a separate search functionality. At least not for now.

**Reply by dlangenk** [reply]
If this is just a few lines of code I would appreciate the search input. Although this doesn't need to be high priority.

**Reply by mzur** [reply]
Then please open an issue for that.
